### DISCUSSION TITLE
Datalink not working

### DISCUSSION THREAD

**Discussion by germanrsolis** [Khopa/dcs_liberation#2617] [Q&A] [answered]
Hi! 
I'm having an issue with the datalink function of my campaign, the datalink fo the SA doesn't seem to be working, after enabling D/L in the F18, no data is shown on the SA page.
Here's the mission, any ideas? (haven't had this issue with other mission in DCS)

Thanks!
[liberation_nextturn.zip](https://github.com/dcs-liberation/dcs_liberation/files/10147886/liberation_nextturn.zip)

**Comment by Chrischan66** [comment]
Have you turned on both D/L?

**Reply by germanrsolis** [reply]
Hey!
Yes, I'm doing that, and pretty much following the same process as always, it's just in these campaigns that the f18 DL doesn't work (f14 works fine, set by Jester)
I'm attaching the last mission where I just had this, it's on the SA map, I can make one in Caucasus if it's easier
[liberation_nextturn.zip](https://github.com/dcs-liberation/dcs_liberation/files/10317509/liberation_nextturn.zip)

**Reply by Chrischan66** [reply]
Hmm, I´ve got D/L working in your mission. do you have specific items missing? bandits or friendly´s?

**Reply by germanrsolis** [reply]
great! it's good to know it's me then haha.
Nothing pops up on the SA screen after enabling both D/L, not even the circle around mi own aircraft, the steps I follow are:
- hit the D/L button
- hit ON (enables the first D/L option)
- hit D/L again
- hit ON again (enables the second D/L option)
Am I missing something 🤔

**Reply by Chrischan66** [reply]
and you hit your sensor-control-switch-aft twice, right?

**Reply by Chrischan66** [reply]
*in case you want to have the information on the center MFD :)

**Reply by germanrsolis** [reply]
Hey! thanks for taking the time to follow up.
I actually use the right MFD, didn't know that pressing the control switch twice aft enables the SA page on the center MFD, thanks! 
Still, it didn't work for me, same behavior. I took a couple of screenshots of another mission on the same campaign:

![Screen_221230_010756](https://user-images.githubusercontent.com/1142127/210035481-a73516a3-3f44-46af-b1b8-b4c10adeaa25.jpg)
![Screen_221230_010800](https://user-images.githubusercontent.com/1142127/210035483-2e5ab613-f2f3-41c3-bfeb-2c81b46d7692.jpg)
![Screen_221230_010806](https://user-images.githubusercontent.com/1142127/210035484-a3c6645d-5c6e-4e88-a5b4-603ffec5533e.jpg)

SA is empty there, I think I enabled it correctly

This is for another mission:

![Screen_221230_012919](https://user-images.githubusercontent.com/1142127/210035521-86c5c8ab-043f-4857-88ca-b6de3c49f538.jpg)
![Screen_221230_012922](https://user-images.githubusercontent.com/1142127/210035522-eabbd59d-53be-4d24-a222-56a0cf0987a6.jpg)

SA page shows the DL contacts, no issues there

I tried repairing the installation and still the same.
Pretty sure I'm missing something obvious, but can't figure out what...

**Reply by Starfire13** [reply]
Hey, just wanted to say that there have been a number of folks who have had issues with datalink in the Hornet in the past. I believe that in all those cases, it turned out to be PEBSAT errors (afaik), and they were all able to get it working eventually. Ultimately, not a Liberation issue.

That said, if you do figure out the issue with your procedure, please do post an update here about where the problem lay because I'm sure you're not going to be the last one with this problem. Best of luck getting it figured out! 🙂

**Reply by germanrsolis** [reply, marked as answer]
hey, thanks! I assume that PEBSAT error is another way to say Layer 8 error, right? hehe, yes, it's probably something like that, I'm continuing that campaign and I'll see if I find what's causing that issue, most likely something I'm just doing wrong or specific to my setup. D/L for other planes, like the F-14 are working just fine on the same campaign, so it shouldn't be something about the mission or how the awacs is set up.

**Reply by Starfire13** [reply]
Problem Exists Between Stick And Throttle, so yeah. Layer 8. 😄

Does datalink work on the Hornet outside of Liberation campaigns?

**Comment by Chrischan66** [comment]
![image](https://user-images.githubusercontent.com/56229370/210059295-06536e40-6dba-4cd6-8410-97a3225e8f10.png)

Don´t know if its important, but I get ALGN in the D/L UFC

**Reply by germanrsolis** [reply]
hey, thanks! yes, I do get that ALGN when I'm on the carrier, still nothing pop ups on the SA page. It must be something specific to this campaign or my setup
